README.rst

```diff
@@ -121,6 +121,26 @@ these exact identifiers directly, if you need something specific.
 The ``source`` release is always automatically included. ``pip`` will use
 this as a fallback in the case a suitable wheel cannot be found.
 
+Dry run mode
+============
+
+There are some use cases, when you maybe don't want to edit your ``requirements.txt``
+right away. You can use the ``-dry-run`` argument to just print the lines, as if they
+would be added to your requirements.txt file.
```

Comment:
I think that this still should have backticks aroud `requirements.txt`. Also, the argument is listed as `-dry-run` with only one `-`, when it should two.

More holistically, didn't this change so that it shows a diff, isntead of "just print[ing] the lines"?